mmv1/products/certificatemanager/TrustConfig.yaml

```diff
@@ -50,8 +50,11 @@ examples:
     primary_resource_id: 'default'
     vars:
       trust_config_name: 'trust-config'
-custom_code: !ruby/object:Provider::Terraform::CustomCode
-  pre_update: templates/terraform/pre_update/certificate_manager_trust_config.go.erb
```

Comment:
Is this pre-update no longer required? It looks like it adds "*" as the update mask, is that no longer needed?